### DISCUSSION TITLE
Unable to set up on Dagster cloud

### DISCUSSION THREAD

**Discussion by MZF404** [dagster-io/dagster#17750] [Q&A] [answered]
Hi, I just started a serverless dagster cloud trial yesterday, and while setting up a sample project, I hit an unexpected critical error (sorry, didn't manage to get a screenshot).

Following that, the setup tabs have disappeared and I'm left with a perpetual spinning icon on my UI. I'm unable to set up with dagster cloud as a result. Is there any fix for this? Thank you!

**Comment by jmsanders** [comment, marked as answer]
It looks like there was a failure when connecting your GitHub repo to Dagster Cloud. I've gone ahead and reset the partial GitHub configuration and have confirmed that the onboarding checklist appears again. Let us know if you bump into any additional issues.

**Reply by MZF404** [reply]
Thank you! It seems like it's working now.
